packages/components/src/item-group/styles.ts

```diff
@@ -6,34 +6,45 @@ import { css } from '@emotion/react';
 /**
  * Internal dependencies
  */
-import { CONFIG, COLORS } from '../utils';
-
-export const unstyledButton = css`
-	appearance: none;
-	border: 1px solid transparent;
-	cursor: pointer;
-	background: none;
-	text-align: start;
-
-	svg,
-	path {
-		fill: currentColor;
-	}
-
-	&:hover {
-		color: ${ COLORS.ui.theme };
-	}
-
-	&:focus-visible {
-		box-shadow: 0 0 0 var( --wp-admin-border-width-focus )
-			var(
-				--wp-components-color-accent,
-				var( --wp-admin-theme-color, ${ COLORS.ui.theme } )
-			);
-		// Windows high contrast mode.
-		outline: 2px solid transparent;
-	}
-`;
+import { CONFIG, COLORS, font } from '../utils';
+
+export const unstyledButton = ( as: 'a' | 'button' ) => {
+	return css`
+		font-size: ${ font( 'default.fontSize' ) };
+		font-family: inherit;
```

Comment:
> Should we also add an explicit font-size to normalize the sizes between button and anchor? ([Like this](https://github.com/WordPress/gutenberg/blob/af38c95e3777b562bdf509647c228c78d68c467a/packages/components/src/dropdown-menu-v2/styles.ts#L162))

I noticed that in addition to font size, the browser default font family was applied to the button element, so I added an inherited style.